.travis.yml

```diff
@@ -47,11 +47,13 @@ dist: trusty
 sudo: required
 addons:
    postgresql: "9.4"
-   apt:
-    packages:
-    - mysql-server-5.6
-    - mysql-client-core-5.6
-    - mysql-client-5.6
+   mysql: "5.7"
```

Comment:
I am currently searching for a way to install mysql without needing root access. This should make travis faster as we can then set `sudo: false`